cvat-ui/src/components/resource-sorting-filtering/filtering.tsx

```diff
@@ -240,16 +240,18 @@ export default function ResourceFilterHOC(
                                 </div>
                             )}
                         >
-                            <Button
-                                className='cvat-quick-filters-button'
-                                type='default'
-                                onClick={() => onPredefinedVisibleChange(!predefinedVisible)}
-                            >
+                            {predefinedFilterValues ? (
```

Comment:
May you please explain this change?
Above there is conditional rendering. If `predefinedFilters` is null, nothing will be rendered

Reply:
It was unnecessary, removed